src/modules/Dashboard/components/ActivityAssignDrawer/ActivityAssignDrawer.tsx

```diff
@@ -245,19 +284,56 @@ export const ActivityAssignDrawer = ({
               />
             </StyledFlexColumn>
 
+            {/* Add respondents */}
             <StyledFlexColumn sx={{ gap: 1.6 }}>
-              <StyledTitleLarge>{t('addRespondents')}</StyledTitleLarge>
+              <StyledFlexTopBaseline sx={{ gap: 1.6 }}>
+                <StyledTitleLarge>{t('addRespondents')}</StyledTitleLarge>
+                {!!assignmentCounts.selfReports && (
+                  <StyledLabelLarge color={variables.palette.on_surface_variant}>
+                    {t('selfReports', { count: assignmentCounts.selfReports })}
+                  </StyledLabelLarge>
+                )}
+                {!!assignmentCounts.multiInformant && (
+                  <StyledLabelLarge color={variables.palette.on_surface_variant}>
+                    {t('multiInformant', { count: assignmentCounts.multiInformant })}
+                  </StyledLabelLarge>
+                )}
+              </StyledFlexTopBaseline>
+
+              <Controller
+                control={control}
+                name="assignments"
+                render={({ field: { onChange, value }, fieldState: { error } }) => {
+                  const duplicateRows =
+                    errors.assignments?.type === 'DuplicateRowsError'
```

Comment:
It would be possible to have validation behave that work, but the current behaviour was part of the AC, as derived from [Figma](https://www.figma.com/design/6BNx63peNVzrHAF1FoZ78x/Multi-Informant-2024?node-id=9404-155473&t=S43bBddJBGbySLgw-4):

![image](https://github.com/user-attachments/assets/28b97b27-5322-402c-9a9d-96753f89865e)